JZCalendarWeekView/JZLongPressWeekView.swift

```diff
@@ -3,28 +3,28 @@
 //  JZCalendarWeekView
 //
 //  Created by Jeff Zhang on 26/4/18.
-//  Copyright © 2018 Jeff Zhang. All rights reserved.
```

Comment:
I don't think this is necessary.

Reply:
it was the xcode formatter, apparently. sure, let me remove it.